MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameEndToEndTests.swift

```diff
@@ -196,7 +206,7 @@ final class MSALNativeAuthSignUpUsernameEndToEndTests: MSALNativeAuthEndToEndBas
         let signUpAttributesRequiredDelegate = SignUpAttributesRequiredDelegateSpy(expectation: submitAttributesExp1)
 
         signUpVerifyCodeDelegate.attributesRequiredNewState?.submitAttributes(
-            attributes: attributes,
+            attributes: ["city": "Dublin"],
```

Comment:
This should take advantage of the `MSAL/test/integration/native_auth/common/AttributesStub.swift` in[ this PR](https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/2261/files) so it should be merged after